example/main.go

```diff
@@ -27,4 +27,5 @@ func main() {
 	if err != nil {
 		// TODO: Handle error
 	}
+	defer resp.Body.Close()
```

Comment:
Wouldn't the `defer` essentially be equal to just calling `resp.Body.Close()` directly? Since it appears to be the last line in the function, so defer would act since the variable itself is no longer in scope?

Reply:
it's just an example, so it's probably doing that so people can paste it and then add more code